### PR TITLE
Add docstrings

### PR DESCRIPTION
**Pull request by tynes**
I went through a single file converting the docstrings to Google Style Python Docstrings.
Let me know if this is along the lines of what you are looking for and I'll continue with the other files.

Will squash into a single signed commit

Could you add a WIP label?

### CODE REVIEW

**Comment by prestwich** [comment]
This is perfect, thanks. Could I convince you to sign your commits?

**Comment by tynes** [comment]
Just pushed up a new signed commit 👍

File: riemann/encoding/cashaddr.py [hunk 1]
```diff
@@ -96,24 +125,52 @@ def calculate_checksum(prefix, payload):
...
     out = list()
     for letter in inputs:
         out.append(CHARSET.find(letter))
     return out
 
 
 def b32encode(inputs):
+    '''
+    Args:
+        inputs ()
```
Comment (tynes) [inline riemann/encoding/cashaddr.py]: TODO: fill this in

**Comment by tynes** [comment]
Will fix the merge conflict + complete the last few files when I have some time. Let me know if you have any feedback

File: riemann/encoding/cashaddr.py (hunk 1 above)
Comment (prestwich) [reply riemann/encoding/cashaddr.py]: I believe inputs should be a list of int

File: riemann/encoding/base58.py [hunk 2]
```diff
@@ -30,7 +30,13 @@
 
 
 def encode(data, checksum=True):
-    """Convert binary to base58 using BASE58_ALPHABET."""
+    """Convert binary to base58 using BASE58_ALPHABET.
```
[the remark this answers is not in the document]
Comment (tynes) [reply riemann/encoding/base58.py]: 👍

**Comment by tynes** [comment]
Just finished rebasing the recent changes

**Comment by tynes** [comment]
Looks like the linter failed, a few lines have too many characters. Will push up a fix when I have time

**Comment by prestwich** [comment]
Closing as stale. Like, 8 months later, because I realized I will never get to it :(

**Pull request closed without merging**